tests/unit/charms/tls_certificates_interface/v4/test_tls_certificates_v4.py

```diff
@@ -400,9 +459,9 @@ def test_given_certificate_string_when_from_string_then_certificate_is_created_c
     )
     certificate_from_string = Certificate.from_string(str(certificate))
     assert certificate_from_string.common_name == "example.com"
-    expected_expiry = datetime.now(timezone.utc) + timedelta(days=200)
+    expected_expiry = datetime.now(timezone.utc) + timedelta(days=200)  # FIXME: Relies on time
     assert abs(certificate_from_string.expiry_time - expected_expiry) <= timedelta(seconds=2)
-    expected_validity_start_time = datetime.now(timezone.utc)
+    expected_validity_start_time = datetime.now(timezone.utc)  # FIXME: Relies on time
```

Comment:
I already have a PR lined up to fix these (and some others in the file), but I'll wait until this is through to make those changes.